management-account/terraform/dsit-integration.tf

```
@@ -0,0 +1,8 @@
+# Infrastructure to integrate cost reporting with DSIT central account: Data export, S3, and bucket replication
+
+module "dsit_infra" {
```

```suggestion
module "dsit_cost_integration" {
```
Maybe use a more descriptive module name